native/src/components/SettingsHeader.tsx

```diff
@@ -32,6 +32,8 @@ const HeaderText = styled.Text`
   flex-direction: column;
   text-align-vertical: center;
   height: 50px;
+  /* fix text-align-vertical for ios, line-height has to be added */
+  line-height: 50px;
```

Comment:
What is the problem without this? Can you add a screenshot please?

Reply:
![image](https://user-images.githubusercontent.com/37902063/151944828-c649d844-ff54-454c-a2ea-c8fe593fff4a.png)